main.c

```diff
@@ -228,7 +228,7 @@ int main(int argc, char** argv) {
     printf("Failed to open audio device: %s\n", SDL_GetError());
     return 1;
   }
-  g_samples_per_block = (534 * g_config.audio_freq) / 32000;
+  g_samples_per_block = (534 * have.freq) / 32000;

```

Comment:
I made this change because technically the user requested config can be incorrect if the device opened by SDL prepares a different sampling rate (I have yet to see this in practice).